comictaggerlib/ui/settingswindow.ui

```diff
@@ -308,197 +336,11 @@
          </item>
         </layout>
        </widget>
-       <widget class="QWidget" name="tComicVine">
+       <widget class="QWidget" name="tComicTalkers">
         <attribute name="title">
-         <string>Comic Vine</string>
+         <string>Comic Sources</string>
```

Comment:
Lets change this to `Metadata Sources` just to make it clear what is being retrieved.